### ISSUE TITLE
Manual Zone Activation Via Panel

### ISSUE THREAD

**Issue by pbezza** [n0bel/ESPrinkler2#5]
Fantastic project. I will start making this in a couple of days. Any chance of adding support for manual activation of zones via the panel? I would love to have buttons on the panel that have configurable on times in minutes (via the web interface) for manual activation of zones. Are you able to add this feature?

**Comment by Meins321** [comment]
Yes manual buttons are missing to test sprinklers and nanually turn on and offf
